config/cable.yml

```diff
@@ -1,5 +1,6 @@
 development:
-  adapter: async
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
This seems like it will require redis in the server environment?

Reply:
i think only if we use something that uses redis (i.e. actioncable), otherwise im assuming this gets ignored